Source/CesiumRuntime/Private/Tests/CesiumFeatureIdTexture.spec.cpp

```diff
@@ -552,7 +552,8 @@ void FCesiumFeatureIdTextureSpec::Define() {
           ECesiumFeatureIdTextureStatus::Valid);
 
       FHitResult Hit;
-      Hit.Location = FVector_NetQuantize::Zero();
+      Hit.Location = FVector_NetQuantize::Zero() *
```

Comment:
Similar nitpick here, any instance of `FVector_NetQuantize::Zero() ` or `FVector_NetQuantize{0, 0, 0}` doesn't need to actually need to use the scale factor